website/docs/cloud-docs/run/cli.mdx

```diff
@@ -27,7 +25,7 @@ Terraform Cloud has three workflows for managing Terraform runs.
 
 The [CLI integration](/cli/cloud) brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It offers the best of both worlds to developers who are already comfortable with using the Terraform CLI, and it can work with existing CI/CD pipelines.
```

Comment:
```suggestion
The [CLI integration](/cli/cloud) brings Terraform Cloud's collaboration features into the familiar Terraform CLI workflow. It stronger functionality developers who are already comfortable with using the Terraform CLI and works with existing CI/CD pipelines.
```

Maybe just a personal thing, but saying "It offers the best of both worlds" feels like marketing speak / overly colloquial language to me. Feel free to ignore if you disagree.

Reply:
this is outside the scope of this PR, so I think I'm going to ignore for now :)